macrosynergy/panel/make_zn_scores.py

```diff
@@ -389,6 +410,8 @@ def make_zn_scores(df: pd.DataFrame, xcat: str, cids: List[str] = None,
 
     if est_freq != "d":
         dfw = dfw.resample(pd_freq[est_freq], axis=0, convention='end').last()
+        # Todo: Fatal bug. This means subsequent applications of pen_neutral etc.
```

Comment:
Is this not the purpose ? To calculate the neutral levels and standard deviations on a lower frequency. For instance, an assert that has a low variance which will subsequently not fluctuate a considerable amount over a week.